pkg/storaged/storage-controls.jsx

```diff
@@ -201,16 +201,16 @@ export const StorageUsageBar = ({ stats, critical, block, offset, total, small }
     const labelText = small ? cockpit.format_bytes(stats[0]) : utils.format_fsys_usage(stats[0], stats[1]);
 
     return (
-        <div className={"pf-c-progress pf-m-outside pf-m-singleline" + (fraction > critical ? " pf-m-danger" : "") + (small ? " pf-m-sm" : "")}>
-            <div className="pf-c-progress__status" aria-hidden="true">
-                <span className="pf-c-progress__measure">{labelText}</span>
+        <div className={"pf-v5-c-progress pf-m-outside pf-m-singleline" + (fraction > critical ? " pf-m-danger" : "") + (small ? " pf-m-sm" : "")}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-161212-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)